reflection/reflection.fbs

```diff
@@ -102,13 +102,22 @@ table Service {
     documentation:[string];
 }
 
+// New schema language features that are not supported by old code generators.
+enum AdvancedFeature : uint {
```

Comment:
use the `bit_flags` attribute?

Reply:
Yea I thought about bit_flags, we'd be limited to 64 new features (I'd rather do u64 than u32). I'm not strongly attached to vector of enums -- I guess 64 flags should be good for a super long time since there's only been 4 since 2014.

Reply:
If we ever hit 64 advanced features we'll add a `advanced_features2` field :P
But seriously, we won't.